tests/TestCase.php

```diff
@@ -2,13 +2,12 @@
 
 namespace Maherelgamil\Arabicdatetime\Tests;
 
-use Maherelgamil\Arabicdatetime\ArabicdatetimeServiceProvider;
 use Orchestra\Testbench\TestCase as OrchestraTestCase;
 
 class TestCase extends OrchestraTestCase
 {
     protected function getPackageProviders($app)
     {
-        return [ArabicdatetimeServiceProvider::class];
+        return ['Maherelgamil\\Arabicdatetime\\ArabicdatetimeServiceProvider'];
```

Comment:
Using the string approach to define the class instance to be compatible with `php-5.4` version.